### DISCUSSION TITLE
Calculation of grey matter volume using charm segmentation

### DISCUSSION THREAD

**Discussion by bronwyn-gavine** [simnibs/simnibs#135] [9. Results and analysis]
Hello simnibs team, 

I'm am trying to calculate tissue volumes from the segmentation of charm results, but unfortunately, as I havent used Freesurfer before, and I'm not sure how much interplay there is between the new segmentation results from charm and freesurfer, I'm struggling to figure it out. I would like to calculate the grey matter, white matter and csf volumes from charm outputs? Specifically, I would like to use the surface data, as my ROI analysis is done using the middle cortical surface. I would be grateful to know if this is easily possible, or if I need to have a separate freesurfer segmentation and surface reconstruction pipeline to do this?

Thanks so much for your help as ever!

Bronwyn

**Comment by oulap** [comment]
Hi Bronwyn,

There's not that much overlap between the output of charm and freesurfer apart from the volume segmentation (labeling.nii.gz). We haven't implemented volume calculation into simnibs, but you could use the labeling to calculate the volumes of the different structures you are interested in (basically just multiply the number of voxels in a given structure with the volume of the voxel). The volumes might not match the freesurfer output exactly but it should be close enough, although the cortical reconstruction in FS is probably more accurate. I'm not entirely sure what you mean by using the surface data: are you thinking about calculating volumes around a specific surface ROI or thickness estimates?

-Oula
